lib/harvesting/client.rb

```diff
@@ -3,7 +3,6 @@
 require "json"
 
 module Harvesting
-
```

Comment:
I'm in favor of removing this empty whitespace but just if we can enforce it using rubocop in our CI